op-node/rollup/sequencing/sequencer.go

```diff
@@ -526,6 +526,11 @@ func (d *Sequencer) startBuildingBlock() {
 		d.log.Info("Sequencing Fjord upgrade block")
 	}
 
+	// For the Fjord activation block we shouldn't include any sequencer transactions.
```

Comment:
@Inphi comment is outdated... did you want to set `attrs.NoTxPool = true` or is this just for logging purposes?

Reply:
We should not set NoTxPool for the granite activation block - there's no upgrade transactions so no reason to exclude L2 transactions. We may have lost the comment update in a cherry-pick somewhere along the line (or I just dreamt that I fixed it at one point...)